modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/TransactionBuilder.kt

```diff
@@ -263,6 +263,19 @@ class TransactionBuilder(builder: TransactionBuilder.() -> Unit = {}) {
         )
     }
 
+    @JvmOverloads
+    fun registerAssetDefinition(
```

Comment:
I would remove this function and construct `AssetDefinitionId` before calling `registerAssetDefinition` function defined above this one

Reply:
we already have the variant you've described
that's the second registerAssetDefinition function in [TransactionBuilder.kt](https://github.com/hyperledger/iroha-java/pull/357/files/264117a0b1fb6171d2c881fdb86dd4f2d511f2ae#diff-7eeee49fa822798628ed15e8a6d47426f7b249ec75d0b62c0327d29b6e749043)

Reply:
> we already have the variant you've described

yes, that's why I thought it looks redundant. Nothing too important, close it if you will